Net02_2/WindowElement.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Net02_2
+{
+    internal class WindowElement
+    {
+        public string Title { get; set; }
+
```

Comment:
Why string Top, Left etc? make integer, not to convert to int each time

Reply:
But I get string value from XML and I will need to do parsing. Isn't my version more optimized?